shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -452,18 +452,22 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
     private predicate notExpectsContent(NodeEx n) { not expectsContentCached(n.asNode(), _) }
 
     pragma[nomagic]
-    private predicate hasReadStep(Content c) { read(_, c, _) }
-
-    pragma[nomagic]
-    private predicate storeEx(
+    private predicate storeEx_(
```

Comment:
Change LGTM, but this predicate name rubs me the wrong way - I don't like ending on an underscore, as it's not really something that we've been doing. I don't really have a good alternative, though.

Reply:
Renamed to `storeExUnrestricted`.